packages/module/package.json

```
@@ -46,7 +46,7 @@
   },
   "peerDependencies": {
     "@patternfly/react-core": "^6.0.0",
-    "react": ">=18.0.0",
+    "react": "17.0.0 || >=18.0.0",
```

this matches what's in react-core:

https://github.com/patternfly/patternfly-react/blob/main/packages/react-core/package.json

```suggestion
    "react": "^17 || ^18",
```